Entities/Student.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class Student : BaseEntity
+    {
+        public List<Timetable> Timetables { get; set; }
```

Comment:
I do not remember why we have 2 timetables for student... But may be it means two attributes not a list

Reply:
OK I change it, now there is only one timetable